README.md

```diff
@@ -32,6 +32,17 @@ A Dockerfile for Sonatype Nexus IQ Server, based on [Red Hat Universal Base Imag
 * [Project License](#project-license)
 
 ## Migration
+
+### Upgrading from Version 100 or Earlier to Version 101 or Later
+Version 1.101.0 of the Docker image changed the base image from Red Hat UBI (Universal Base Image) to include
```

Comment:
changing **from** an image implies you are changing **to** something else, when it sounds like all we did was include OpenJDK into the base image?

Reply:
point is the wording here is confusing me what was actually done and even "why" it was done

Reply:
> changing **from** an image implies you are changing **to** something else, when it sounds like all we did was include OpenJDK into the base image?

It's a different image
old image is registry.access.redhat.com/ubi8/ubi
https://catalog.redhat.com/software/containers/ubi8/ubi/5c359854d70cc534b3a3784e

new image is registry.access.redhat.com/ubi8/openjdk-8
https://catalog.redhat.com/software/containers/ubi8/openjdk-8/5dd6a48dbed8bd164a09589a

Tried to clarify in https://github.com/sonatype/docker-nexus-iq-server/pull/52/commits/3d752299ae02f8fa3741ffe37f9429976d7e6780

Reply:
> point is the wording here is confusing me what was actually done and even "why" it was done

Included why it was changed https://github.com/sonatype/docker-nexus-iq-server/pull/52/commits/d8dc6019c924deed0c16f7d94346b6998aecdcd6 (remove dependency on chef)

Reply:
> It's a different image

is there harm in just including these links to the images as part of the readme?

Reply:
> is there harm in just including these links to the images as part of the readme?

Don't see any harm added in https://github.com/sonatype/docker-nexus-iq-server/pull/52/commits/b1b1efa45b4a0b5b2809864b5be00543c50986ae

Reply:
Can we get this published soon? We've had two customers with failed upgrades because of it.  We should also call this out in the release notes.

Reply:
> Can we get this published soon? We've had two customers with failed upgrades because of it. We should also call this out in the release notes.

Created a release note (essentially a copy of the one we had for release 69)
https://help.sonatype.com/display/NXIQM/2020+Release+Notes
https://help.sonatype.com/pages/diffpagesbyversion.action?pageId=43352397&selectedPageVersions=243&selectedPageVersions=242

https://help.sonatype.com/display/NXIQM/Upgrading+the+IQ+Server
https://help.sonatype.com/pages/diffpagesbyversion.action?pageId=330916&selectedPageVersions=23&selectedPageVersions=22